server/datastore/mysql/activities.go

```diff
@@ -286,18 +286,18 @@ func (ds *Datastore) ListHostUpcomingActivities(ctx context.Context, hostID uint
 		// list pending scripts
 		`SELECT
 			ua.execution_id as uuid,
-			IF(ua.fleet_initiated, 'Fleet', COALESCE(u.name, JSON_EXTRACT(ua.payload, '$.user.name'))) as name,
+			IF(ua.fleet_initiated, 'Fleet', COALESCE(u.name, ua.payload->>'$.user.name')) as name,
```

Comment:
Note (TIL): `->>` is `JSON_UNQUOTE(JSON_EXTRACT(col, path))`, which is necessary for string values otherwise it would return e.g. `"username"`, with the double-quotes as part of the value. So I went ahead and changed the uses of `JSON_EXTRACT` to `->>` for string values and `->` (equivalent to JSON_EXTRACT) for other types.